packages/identity-app/src/ManageAddresses/ManageAddresses.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { Dropdown, Menu } from '@substrate/ui-components';
+import React from 'react';
+import { Link, Route, RouteComponentProps, Switch } from 'react-router-dom';
+
+import { SaveAddress } from './SaveAddress';
+import { MatchParams } from '../types';
+
+interface Props extends RouteComponentProps<MatchParams> { }
+
+const menuOptions = [
+  {
+    key: 'Add',
+    text: 'Add',
+    value: 'Add'
+  }
+];
+
+export class ManageAddresses extends React.PureComponent<Props> {
+  render () {
+    return (
+      <React.Fragment>
+        {this.renderMenu()}
+        <Switch>
+          {/* FIXME add route for Edit component */}
```

Comment:
@yjkimjunior I wasn't sure here, are you planning to have 2 components, one to create and one to edit? Or SaveAddress should do both?

Reply:
@amaurymartiny yes my bad I forgot the edit component for ManageAddresses (should more or less be the same as from ManageAccounts). 
i.e:
ManageAccounts -> Create/Edit/Restore
ManageAddresses -> Add/Edit